map/src/main/res/layout/fragment_search_object_details.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file will need to act more like a carousel in future, right? If not, you should really extract some common styles and compact down this file.

Reply:
You are right. This fragment will have carousel logic in the future.